### PR TITLE
BUG Fix enable email subclasses to use their respective templates

### PR DESCRIPTION
**Pull request by flamerohr**
Fixes https://github.com/silverstripe/silverstripe-framework/issues/7470

### CODE REVIEW

**Comment by tractorcow** [comment]
I fixed this and updated tests AND I fixed all those old warnings on EmailTest. :)

**Comment by tractorcow** [comment]
Oh, the test wasn't the one you wanted @kinglozzer. Sorry I'll add it now. :)

File: src/View/ThemeResourceLoader.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function getPath($identifier)
         }
 
         // Join module with subpath
-        return $modulePath . $subpath;
+        return ltrim($modulePath . $subpath, '/');
```
Comment (tractorcow) [inline src/View/ThemeResourceLoader.php]: Fixes a bug where $modulePath is empty but $subPath has a leading `/`

**Comment by flamerohr** [comment]
Nice, thanks for the additional fixes and unit tests @tractorcow 
they look good to me

**Comment by kinglozzer** [comment]
Nice job, thanks both!

**Pull request merged**